examples/prom/nonlinear_elasticity_global_rom.cpp

```diff
@@ -0,0 +1,1747 @@
+            int ctr = 0;
+            for (size_t i = 0; i < sp_size; i++)
+            {
+                if (Ess_mat_sp(i,0) == 1)
+                {
+                    ess_tdof_list_sp[ctr] = i;
+                    ctr += 1;
+                }
+
+            }
+
+            // Define operator in sample space
+            soper = new HyperelasticOperator(*sp_XV_space, ess_tdof_list_sp, visc, mu, K);
+
+        }
+
+        if (hyperreduce)
+        {   romop = new RomOperator(&oper, soper, rvdim, rxdim, hdim, smm, *w_v0, *w_x0,
+                                    vx0.GetBlock(0), BV_librom, BX_librom, H_librom, Hsinv, myid,
+                                    num_samples_req != -1, hyperreduce, x_base_only);
+        }
+        else
+        {
+            romop = new RomOperator(&oper, soper, rvdim, rxdim, hdim, smm, vx0.GetBlock(0),
+                                    vx0.GetBlock(1), vx0.GetBlock(0), BV_librom, BX_librom, H_librom, Hsinv, myid,
+                                    num_samples_req != -1, hyperreduce, x_base_only);
+        }
+
+        // Display lifted initial energies
+        BroadcastUndistributedRomVector(w);
+
+        for (int i=0; i<rvdim; ++i)
+            (*w_v)(i) = (*w)(i);
+
+        for (int i=0; i<rxdim; ++i)
+            (*w_x)(i) = (*w)(rvdim + i);
+
+        romop->V_v.mult(*w_v, *v_rec_librom);
+        romop->V_x.mult(*w_x, *x_rec_librom);
+
+        *v_rec += vx0.GetBlock(0);
+        *x_rec += vx0.GetBlock(1);
+
+        v_gf.SetFromTrueDofs(*v_rec);
+        x_gf.SetFromTrueDofs(*x_rec);
+
+        double ee = oper.ElasticEnergy(x_gf);
+        double ke = oper.KineticEnergy(v_gf);
+
+        if (myid == 0)
+        {
+            cout << "Lifted initial energies, EE = " << ee
+                 << ", KE = " << ke << ", ΔTE = " << (ee + ke) - (ee0 + ke0) << endl;
+        }
+
+        ode_solver->Init(*romop);
+    }
+    else
+    {
+        // fom
+        ode_solver->Init(oper);
+    }
+
+    // 13. Perform time-integration
+    //     (looping over the time iterations, ti, with a time-step dt).
+    //     (taking samples and storing it into the pROM object)
+
+    double t = 0.0;
+    vector<double> ts;
+    oper.SetTime(t);
+
+    bool last_step = false;
+    for (int ti = 1; !last_step; ti++)
+    {
+        double dt_real = min(dt, t_final - t);
+
+        if (online)
+        {
+            if (myid == 0)
+            {
+                solveTimer.Start();
+                ode_solver->Step(*wMFEM, t, dt_real);
+                solveTimer.Stop();
+            }
+
+            MPI_Bcast(&t, 1, MPI_DOUBLE, 0, MPI_COMM_WORLD);
+
+        }
+        else
+        {
+            solveTimer.Start();
+
+            ode_solver->Step(vx, t, dt_real);
+
+            solveTimer.Stop();
+
+        }
+
+        last_step = (t >= t_final - 1e-8 * dt);
+
+        if (offline)
+        {
+
+            if (  basis_generator_x->isNextSample(t) || x_base_only == false && basis_generator_v->isNextSample(t))
```

Comment:
Extra spaces. Astyle may fix this automatically.

Reply:
That's true. I'll run Astyle again